homeassistant/components/ecovacs/config_flow.py

```diff
@@ -77,19 +95,25 @@ async def async_step_user(
                                 type=selector.TextSelectorType.PASSWORD
                             )
                         ),
-                        vol.Required(CONF_COUNTRY): vol.All(vol.Lower, cv.string),
-                        vol.Required(CONF_CONTINENT): vol.All(vol.Lower, cv.string),
```

Comment:
Removing the continent requires no migration, as no stable release included the config flow of ecovacs